DRAFT.md

```diff
@@ -282,6 +286,9 @@ class indirect {
   template <class U>
   friend constexpr auto operator<=>(const U& lhs, const indirect<T, A>& rhs);
 };
+
+template <class T, class Alloc>
+struct std::uses_allocator<indirect<T>, Alloc> : true_type {};
```

Comment:
Q: Should this be `indirect<T, Alloc>` ?

Reply:
Good catch, now corrected